gateways/http/account/get_balance.go

```diff
@@ -13,13 +13,14 @@ type AccountBalance struct {
 
 // ShowAccount godoc
```

Comment:
probably you've kept this line because of [swago](https://github.com/swaggo/swag) documentation, but this just represents the first line of your handler, as a method, documentation. Therefore, `ShowAccount` doesn't fit well here and amid other handlers